contrib/install-dependencies.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+apt update
+apt install genisoimage mtools python3-distro-info -y
```

Comment:
I'd like to make this more distro-agnostic overtime, so I'd suggest we print a warning that only Debian/Ubuntu are supported if we find we're not on one of those systems. We could then enable more OS's over time. Also seems like we could put this at the top level in e.g. a `./setup.sh` or similar instead of in a `contrib/` subdir.

Reply:
That's great! I am thinking the same thing as well. Let me file a feature request for the implementation of dependency checker. See https://github.com/dannf/ubuntu-server-netboot/issues/12 .

Reply:
I prefer to implement this later on, and focus on `--autoinstall-url` instead. What do you think?

Reply:
Either order works for me, thanks!